Makefile

```diff
@@ -0,0 +1,53 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+.PHONY: build
+ARCH ?= amd64
+QEMUVERSION=v2.9.1
+TEMP_DIR:=$(shell mktemp -d)
+BUILD_IMAGE ?= wurstmeister/kafka-docker:1.0.0
+
+ifeq ($(ARCH),amd64)
+        BASEIMAGE?=anapsix/alpine-java
+endif
+ifeq ($(ARCH),ppc64le)
+	BASEIMAGE?=openjdk:jdk-alpine
+        QEMUARCH=ppc64le
+        BUILD_IMAGE = wurstmeister/kafka-docker-$(ARCH):1.0.0
+endif
+
+build: 
+
+ifeq ($(ARCH),amd64)
+	cp ./* $(TEMP_DIR)
+	cat Dockerfile.crossbuild \
+                | sed "s|BASEIMAGE|$(BASEIMAGE)|g" \
+                | sed "/CROSS_BUILD_COPY/d" \
+                > $(TEMP_DIR)/Dockerfile.crossbuild
+	# We just build it using the usual process.
+	docker build -t $(BUILD_IMAGE) -f $(TEMP_DIR)/Dockerfile.crossbuild $(TEMP_DIR)
```

Comment:
How about replacing `crossbuild` with `ARCH` (since it's no longer crossbuild)?

Reply:
Are you suggesting changing the name of the file?
Ideally I kept one file to avoid any out of sync issues across ARCHs. 
Another way could be that I write `> $(TEMP_DIR)/Dockerfile.ARCH` instead of `>$(TEMP_DIR)/Dockerfile.crossbuild` and then build using that file.  Thoughts?